sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/models/ChatRetentionPolicy.java

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
It's not necessarily an issue to have public models that map to impl versions. That said, I think you should consider using AutoRest transforms to make some of those impl models the public-facing code and avoid duplication in the future. You can find an example of how we do this for the Key Vault library [here](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/keyvault/azure-security-keyvault-certificates/swagger/autorest.md).